docs/en/tutorials/composite-primary-keys.rst

```diff
@@ -14,12 +14,12 @@ General Considerations
 Every entity with a composite key cannot use an id generator other than "NONE". That means
 the ID fields have to have their values set before you call ``EntityManager#persist($entity)``.
 
-Primitive Types only
-~~~~~~~~~~~~~~~~~~~~
+Allowed Types
+~~~~~~~~~~~~~
 
 You can have composite keys as long as they only consist of the primitive types
```

Comment:
You changed the title probably because you're considering `DateTimeImmutable` isn't a primitive type. Should this sentence be changed as well then?

Reply:
Whole sentence is 
> You can have composite keys as long as they only consist of the primitive types ``integer`` and ``string`` **or** object of ``DateTimeImmutable``